spec/factories/comments.rb

```diff
@@ -3,5 +3,10 @@
     user
     article
     sequence(:body) { |b| "Hello, World!-#{b}" }
+
+    before(:create) do |f|
+      f.article_id = f.article.try(:id)
```

Comment:
Do we really need it?

Reply:
there are associations already in factory and they should be automatically assigned